Test-Source/Invoke-ExecCommandCore.Tests.ps1

```diff
@@ -0,0 +1,87 @@
+Import-Module -Force $PSScriptRoot/../Source/Docker.Build.psm1
+
+. "$PSScriptRoot\..\Source\Private\Invoke-ExecCommandCore.ps1"
+. "$PSScriptRoot\..\Source\Private\CommandCoreResult.ps1"
+
+Describe 'Runs only external tools' {
```

Comment:
Consider adding test cases for when there are:

1. Zero args
2. Multiple args.

Reply:
Have that on locally. Still WIP tbh. :sweat_smile: